src/snowflake/snowpark/modin/pandas/dataframe.py

```diff
@@ -666,7 +666,6 @@ def add(
             fill_value=fill_value,
         )
 
-    @dataframe_not_implemented()
     def assign(self, **kwargs):  # noqa: PR01, RT01, D200
         """
         Assign new columns to a ``DataFrame``.
```

Comment:
Can you update the doc string in `src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py`?